apps/aehttp/src/aehttp_api_encoder.erl

```diff
@@ -168,6 +168,8 @@ type2enc(block_state_hash) -> ?BASE58;
 type2enc(channel)          -> ?BASE58;
 type2enc(contract_pubkey)  -> ?BASE58;
 type2enc(contract_bytearray)-> ?BASE64;
+type2enc(contract_store_key)-> ?BASE64;
```

Comment:
Adjust spaces so the lines don't stand out.

Reply:
I was done the same as existing code.